Makefile

```diff
@@ -87,7 +87,7 @@ test:
 # Helm charts generation and templates processing
 ################################################################################
 
-SUPPORTED_VERSIONS := 2.0 2.1 2.2 2.3 2.4
+SUPPORTED_VERSIONS := 2.1 2.2 2.3 2.4
```

Comment:
Shouldn't 2.1 be removed from here as well?

Reply:
Oh, I forgot about it.

Reply:
We decided to not remove 2.1 related code entirely, so I will not change this line.